src/pages/task/Metadata.tsx

```diff
@@ -39,29 +42,46 @@ export const Metadata: React.FC<{
   const author = patchMetadata?.author;
 
   const distroLink = `${getUiUrl()}/distros##${distroId}`;
+
   return (
     <MetadataCard error={error} loading={loading} title="Task Metadata">
       <P2>Submitted by: {author}</P2>
       <P2 data-cy="task-metadata-submitted-at">
         Submitted at: {getDateCopy(createTime)}
       </P2>
-      <P2>
-        Estimated time to start:{" "}
-        <span data-cy="task-metadata-estimated_start">
-          {msToDuration(estimatedStart)}
-        </span>
-      </P2>
-      <P2>
-        Started:{" "}
-        <span data-cy="task-metadata-started">{getDateCopy(startTime)}</span>
-      </P2>
-      <P2 data-cy="task-metadata-finished">
-        Finished:{" "}
-        <span data-cy="task-metadata-started">{getDateCopy(finishTime)}</span>
-      </P2>
-      <P2 data-cy="task-metadata-duration">
-        Duration: {msToDuration(timeTaken)}{" "}
-      </P2>
+      {estimatedStart > 0 && (
+        <P2>
+          Estimated time to start:{" "}
+          <span data-cy="task-metadata-estimated_start">
+            {msToDuration(estimatedStart)}
+          </span>
+        </P2>
+      )}
+      {/* Can only show the time running and eta if the task is running and 
+      it has a baseTaskDuration to calculate the eta with */}
+      {status === TaskStatus.Started && baseTaskDuration && (
+        <P2>
+          Running Time / ETA:{" "}
```

Comment:
nit: I think ETA refers to a date but we are actually showing a duration. I think "ETA" should actually be "Time Left"